app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -2,6 +2,11 @@
 
 module ExceptionHunter
   class ErrorsController < ApplicationController
+    helper_method :dashboard_user
+    before_action do
+      send("authenticate_#{dashboard_user}!") if dashboard_user
```

Comment:
We can just generate this in the initializer, it's easier for people to override it in case they want some custom config.